virtual_rainforest/models/soil/constants.py

```diff
@@ -4,117 +4,141 @@
 """  # noqa: D205, D415
 
 from dataclasses import dataclass
-from typing import Final
 
 # TODO - Need to figure out a sensible area to volume conversion
 
 
 @dataclass(frozen=True)
-class BindingWithPH:
-    """From linear regression :cite:p:`mayes_relation_2012`."""
+class SoilConsts:
+    """Dataclass to store all constants for the `abiotic_simple` model."""
 
-    slope: float = -0.186
-    """Units of pH^-1."""
-    intercept: float = -0.216 + 3.0
-    """Unit of log(m^3 kg^-1). n.b. +3 converts from mg^-1 to kg^-1 and L to m^3"""
+    binding_with_ph_slope: float = -0.186
+    """Units of [pH^-1]. From linear regression :cite:p:`mayes_relation_2012`."""
```

Comment:
Could you add a bit more information about the parameters, especially the ones from linear regression?

Reply:
I've added extra details for the parameters from linear regression.

For the temperature and moisture scalars, I'm honestly not completely sure what they are. I've added a TODO about this because it's a much bigger problem that the description of constants